<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Посещение</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_students">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>621</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id_students</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Класс</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ID_parents</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_Run">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_parents">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>621</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id_parents</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчество</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_movements">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>621</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id_students</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TIME_IN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TIME_OUT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Присутствие</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_add_student">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить ученика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del_student">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить ученика</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_event">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить событие</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="separator"/>
    <addaction name="period"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Период наблюдения</string>
   </property>
  </action>
  <action name="period">
   <property name="text">
    <string>Период наблюдения</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>ячсмчав</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
